tests/shell/tests/01-run.py

```diff
@@ -27,6 +27,9 @@
     '\t  \d | running  Q |   7'
 )
 
+RIOT_TERMINAL = os.environ.get('RIOT_TERMINAL')
+CLEANTERMS = { "cleanterm", "socat" }
```

Comment:
`cleanterm` is used already for all tests, so it should just be socat

Reply:
fixed

Reply:
Did you forget to push?